qiskit_ibm_runtime/api/clients/runtime.py

```diff
@@ -54,7 +53,6 @@ def program_run(
         backend_name: Optional[str],
         params: Dict,
         image: Optional[str],
-        hgp: Optional[str],
```

Comment:
The `RuntimeClient` class is not part of the public API technically but it is referenced from other API reference entries, for example https://quantum.cloud.ibm.com/docs/en/api/qiskit-ibm-runtime/runtime-job-v2. In the future we should do a review of the api reference.